pdns/pdns_recursor.cc

```diff
@@ -730,11 +730,8 @@ void startDoResolve(void *p)
     if(!g_quiet || tracedQuery) {
       L<<Logger::Warning<<t_id<<" ["<<MT->getTid()<<"/"<<MT->numProcesses()<<"] " << (dc->d_tcp ? "TCP " : "") << "question for '"<<dc->d_mdp.d_qname<<"|"
        <<DNSRecordContent::NumberToType(dc->d_mdp.d_qtype)<<"' from "<<dc->getRemote();
-#ifdef HAVE_PROTOBUF
```

Comment:
@rgacogne Judging by our IRC convo on ECS a few days ago, this will break when compiling without on non-protobuf. Could you revert this commit?